models/issue_watch.go

```diff
@@ -98,35 +145,45 @@ func getIssueWatchers(e Engine, issueID int64, listOptions ListOptions) (watches
 
 func removeIssueWatchersByRepoID(e Engine, userID int64, repoID int64) error {
 	iw := &IssueWatch{
-		IsWatching: false,
+		Mode: IssueWatchModeNone,
 	}
 	_, err := e.
 		Join("INNER", "issue", "`issue`.id = `issue_watch`.issue_id AND `issue`.repo_id = ?", repoID).
-		Cols("is_watching", "updated_unix").
+		Cols("mode", "updated_unix").
 		Where("`issue_watch`.user_id = ?", userID).
 		Update(iw)
 	return err
 }
 
-// LoadWatchUsers return watching users
-func (iwl IssueWatchList) LoadWatchUsers() (users UserList, err error) {
-	return iwl.loadWatchUsers(x)
+// IsWatching is true if user iw watching either repo or issue (backwards compatibility)
+func (iw IssueWatch) IsWatching() bool {
+	issue, err := GetIssueByID(iw.IssueID)
+	if err != nil {
+		// fail silent since template expect only bool
+		log.Error("IssueWatch.IsWatching: GetIssueByID: ", err)
```

Comment:
```suggestion
		log.Warn("IssueWatch.IsWatching: GetIssueByID: ", err)
```
IMHO `Error` should be used for aborted operations; `Warn` otherwise.